tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```diff
@@ -53,6 +53,23 @@ Services:
       Java: noop
       JS: NoOp
 
+- Service: core
+  Languages:
+    Java:
+      Project: sdk/core/azure-core-perf
+      PackageVersions:
+      - azure-core: 1.29.1
+      - azure-core: source
```

Comment:
we should also list `azure-core-http-netty` and `azure-core-http-okhttp`

See
https://github.com/Azure/azure-sdk-for-java/blob/61979cc179df3127cafbf58abc0e41090291a53c/sdk/core/azure-core-perf/pom.xml#L26-L35